### ISSUE TITLE
Team review of gslab_scons module

### ISSUE THREAD

**Issue by lboxell** [gslab-econ/gslab_python#16]
Follow up to https://github.com/gslab-econ/template/issues/16. Per https://github.com/gslab-econ/admin/issues/67#issuecomment-243509241 and https://github.com/gslab-econ/template/pull/7#issuecomment-243500575, we would like to have everyone review the gslab_scons module as time permits.

After https://github.com/gslab-econ/template/issues/16, everyone should be able to download the gslab_scons module as a part of the gslab_tools package on pypi and use gslab_scons module within the template repo.

Probably low priority task relative to finalizing TRANS and congress_text, and @gentzkow can decide how he would like to aggregate comments (either discussion in person or via GitHub). 

Also, flagging @jmshapir because he should be able to begin playing around with the module at this point and add any other comments he wants to regarding the code.

Evaluation can be done in parallel with https://github.com/gslab-econ/template/issues/19.
